### PR TITLE
Some profiles

### PR DESCRIPTION
**Pull request by nyancat18**
* Fixed Profiles: Kdenlive, Shotcut

* New Profile: Cinelerra, Cliqz, Bluefish

* Some hardening for some profiles

### CODE REVIEW

File: etc/calligra.profile [hunk 1]
```diff
@@ -21,9 +21,10 @@ novideo
 protocol unix
 seccomp
 shell none
+net none
 
 private-bin calligra,calligraauthor,calligraconverter,calligraflow,calligraplan,calligraplanwork,calligrasheets,calligrastage,calligrawords,dbus-launch
 private-dev
 
-noexec ${HOME}
+#noexec ${HOME}
```
Comment (SpotComms) [inline etc/calligra.profile]: What does calligra need to execute? Are there extensions/plugins?

File: etc/tar.profile [hunk 2]
```diff
@@ -18,6 +18,7 @@ notv
 novideo
 shell none
 tracelog
+caps.drop all
```
Comment (reinerh) [inline etc/tar.profile]: are the capabilities not required for setting certain file flags, owner etc.?

**Comment by SpotComms** [comment]
- 7z, tar, unrar, and unzip include default.profile so adding `caps.drop all` isn't needed
- evince having `net none` is known to cause issues see https://github.com/netblue30/firejail/pull/1367#issuecomment-315770405
- Instead of creating a profile for openshot-qt why not create an alias/redirect to openshot.profile
- Also your ~/.gitconfig doesn't have your user/email set
- Could you please try and stick to the format and keep options in alphabetical order

File: etc/atom.profile [hunk 3]
```diff
@@ -23,6 +25,7 @@ notv
 novideo
 protocol unix,inet,inet6,netlink
 seccomp
+net none
```
Comment (reinerh) [inline etc/atom.profile]: does allowing inet(6) protocols still make sense with "net none"?

Comment (SpotComms) [reply etc/atom.profile]: No, it would also break the extension/themes store

File: etc/7z.profile [hunk 4]
```diff
@@ -17,6 +17,7 @@ notv
 novideo
 shell none
 tracelog
+caps.drop all
```
Comment (reinerh) [inline etc/7z.profile]: are some capabilities not required for setting certain flags, file ownership etc.?

File: etc/atom.profile (hunk 3 above)
Comment (nyancat18) [reply etc/atom.profile]: we could put it optional :)

**Comment by reinerh** [comment]
@SpotComms I wasn't aware that default.profile also drops caps.
I just confirmed that it breaks archiving tools run as root...
I think the setting is too invasive. Other opinions?

```
# firejail tar xf foo.tar 
tar: foo: Cannot change ownership to uid 1000, gid 1000: Operation not permitted
tar: Exiting with failure status due to previous errors
```

**Comment by SpotComms** [comment]
@reinerh I'm actually surprised that hasn't been caught earlier. We can either add `ignore caps.drop` to them all or switch them all from default.profile to a generic compression one, the latter might be better.

**Comment by Fred-Barclay** [comment]
@SpotComms @reinerh I rather like the idea of a generic compression profile. 👍

File: etc/7z.profile (hunk 4 above)
Comment (netblue30) [reply etc/7z.profile]: removed caps.drop all

File: etc/atom.profile (hunk 3 above)
Comment (netblue30) [reply etc/atom.profile]: optional net none (commented out)

File: etc/evince.profile [hunk 5]
```diff
@@ -28,6 +28,7 @@ protocol unix
 seccomp
 shell none
 tracelog
+net none
```
Comment (netblue30) [inline etc/evince.profile]: commented out, breaks AppArmor on Ubuntu systems

File: etc/tar.profile (hunk 2 above)
Comment (netblue30) [reply etc/tar.profile]: removed caps.drop all

**Comment by netblue30** [comment]
Merged, thanks.

I'll bring some fixes in, I'll go with the list from @SpotComms: https://github.com/netblue30/firejail/pull/1565#issuecomment-330705836

And yes, we have to figure out some generic profile for compression tools!

**Comment by SpotComms** [comment]
@netblue30 already fixed in def6f1ee2435a4ce4712fdd709ea9d14253547f0 and 04adc450151cc5107098ef2f555ad526ac9f632e
I think I addressed everything and a few others.

**Comment by netblue30** [comment]
Cool! I also added some small fixes.

File: etc/evince.profile (hunk 5 above)
Comment (nyancat18) [reply etc/evince.profile]: could  you  add a comment?.

just like.

>#please uncomment if you dont have apparmor *archlinux*

**Pull request merged**